cca_example_test.go

```diff
@@ -0,0 +1,164 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix/mat64"
+	"github.com/gonum/stat"
+)
+
+// symView is a helper for getting a View of a SymDense.
+type symView struct {
```

Comment:
mat64.SymDense.SquareSlice doesn't do what I need here -- which is to take a non-symmetric submatrix of a SymDense. mat64.SymDense.SquareSlice only allows for symmetric submatrices (along the diagonal). Or did I misunderstand?